integration_tests/test_basic.py

```diff
@@ -949,6 +950,77 @@ def test_submit_send_enabled(cronos, tmp_path):
     assert cli.query_bank_send(*denoms) == send_enable
 
 
+def test_join_validator(cronos):
```

Comment:
is this test case related to the sdk fix?

Reply:
maybe can revert, since it was to avoid accidentally include decimal [change](https://github.com/cosmos/cosmos-sdk/pull/21825/files)